NDcPP_v3_0.adoc

```diff
@@ -3003,15 +2995,15 @@ _The ST author must include FIA_X509_EXT.1/Rev in all instances except when only
 
 *FIA_X509_EXT.2 X.509 Certificate Authentication*
 
-*FIA_X509_EXT.2.1* The TSF shall use X.509v3 certificates as defined by RFC 5280 to support authentication for [selection: _DTLS, HTTPS, IPsec, SSH, TLS, no protocols_] and [selection: _code signing for system software updates[assignment: other uses], no additional uses_].
+*FIA_X509_EXT.2.1* The TSF shall use X.509v3 certificates as defined by RFC 5280 to support authentication for [selection: _DTLS, HTTPS, IPsec, TLS, SSH, [assignment: other protocols], and [selection: _code signing for system software updates [assignment: other uses], no additional uses_].
 
 *FIA_X509_EXT.2.2* When the TSF cannot establish a connection to determine the validity of a certificate, the TSF shall [selection: _allow the Administrator to choose whether to accept the certificate in these cases, accept the certificate, not accept the certificate_].
 
 *_Application Note {counter:appnote_count}_*
 
 _In FIA_X509_EXT.2.1, the ST author’s selection includes IPsec, (D)TLS, or HTTPS if these protocols are included in FTP_ITC.1.1 or FPT_ITT.1. The ST author selects “code signing for system software updates” when “X.509 certificate” is selected in FPT_TUD_EXT.1.3._
 
-_SSH should be included if the TSF shall be validated against the version of the Functional Package for Secure Shell referenced in section 2.2, and authentication other than ssh-rsa, ecdsa-sha2-nistp256, ecdsa-sha2-nistp384, and/or ecdsa-sha2-nistp521 is selected._
+_SSH should be included if the TSF should be validated against the version of the Functional Package for Secure Shell referenced in section 2.2, and authentication other than ssh-rsa, ecdsa-sha2-nistp256, ecdsa-sha2-nistp384, and/or ecdsa-sha2-nistp521 is selected._
```

Comment:
If X.509 is supported/claimed in SSH then FIA_X509 is required, right? If so; suggested modification: 

'SSH shall be included if the TSF is to be validated against the version of the Functional Package for Secure Shell referenced in section 2.2, and X.509-based authentication is selected in that Functional Package.'